.github/workflows/pr.yml

```diff
@@ -26,20 +26,20 @@ jobs:
         env:
           cache-name: node-modules
         with:
-          path: ~/.npm
+          path: ~/.yarn
           key: ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
```

Comment:
This should look at `yarn.lock` to ensure the cache is set correctly.

Reply:
@jerelmiller I can fix that in a new PR.

Reply:
Updated this in another PR https://github.com/newrelic/opensource-website/pull/733